Lesson22/Anil/todo-app/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>To-Do App</title>
+		<link rel="stylesheet" type="text/css" href="styles.css" />
+	</head>
+	<body>
+
+		<button id="btnSayHello">Say hello</button>
+		<button id="btnSayBye">Say bye</button>
+
+		<div class="container">
+			<h1>To-Do List</h1>
+
+			<form id="toDoForm">
+				<input type="text" placeholder="Enter to do" required id="toDoInput"/>
+				<button type="submit" >submit</button>
+				<button type="reset">reset</button>
+			</form>
+
+			<ul id="todoList">
+				<li>New todo example<button>X</button></li>
```

Comment:
Let's remove this placeholder element as it does not work as other <li> elements so it's a bit misleading.

Reply:
I removed it